index.bs

```diff
@@ -515,9 +516,8 @@ The abstract operation LoadModule with arguments <i>entry</i> and <i>stage</i> p
     1. Return *undefined*.
 1. If _stage_ is "link", then:
   1. Return the result of transforming RequestLink(_entry_) with a fulfillment handler that returns *undefined*.
-1. If _stage_ is "ready" or *undefined*, then:
-  1. Return the result of transforming RequestReady(_entry_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
-    1. Return GetModuleNamespace(_entry_.[[Module]]).
+1. If _stage_ is "ready", then:
+  1. Return the result of transforming RequestReady(_entry_) with a new pass-through promise.
```

Comment:
this is the only place where we now call RequestReady(). Moved the call to GetModuleNamespace() into it.